lumigator/schemas/lumigator_schemas/jobs.py

```diff
@@ -224,9 +224,9 @@ class JobResultObject(BaseModel):
     """
 
     model_config = ConfigDict(extra="forbid")
-    metrics: dict = {}
-    parameters: dict = {}
-    artifacts: dict = {}
+    metrics: dict = Field(default_factory=dict)
```

Comment:
What's the advantage of using a factory here?

Reply:
Ah TIL Pydantic handles the mutable objects for you (https://docs.pydantic.dev/latest/concepts/fields/#mutable-default-values). I can revert this change since it makes no difference.

Reply:
Hmmmm, please send the link for those other examples, we may need to simplify those a bit. Thanks!!!!

Reply:
We need a LMGTFY but 'let me GitHub that for you' (LMGHTFY) 😆 

https://github.com/mozilla-ai/lumigator/blob/22692469e6f59aa95163ba524ed58a46b4aaaf24/lumigator/schemas/lumigator_schemas/jobs.py#L68-L74